Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.2
+MAINTAINER dengnan@google.com vmarmol@google.com vishnuk@google.com jimmidyson@gmail.com
+
+WORKDIR /gopath/src/github.com/google/cadvisor
+COPY    . /gopath/src/github.com/google/cadvisor
+
+RUN apk add --update --repository=http://dl-4.alpinelinux.org/alpine/edge/community/ -t build-deps go sqlite-dev btrfs-progs-dev bash linux-headers make git gcc g++ && \
+    apk add --update device-mapper && \
+    export GOPATH=/gopath && \
```

Comment:
Because these env vars aren't needed after the image build & all go build tools are cleaned up anyway. Not expecting any images to extend this one & build go stuff.